include/picolibrary/wiznet/w5500.h

```diff
@@ -486,6 +486,1219 @@ class Communication_Controller : public Device {
     }
 };
 
+/**
+ * \brief Mode Register (MR) register.
+ *
+ * This register has the following fields:
+ * - Force ARP Request Enable (FARP)
+ * - PPPoE Enable (PPPOE)
+ * - Ping Block Enable (PB)
+ * - WoL Enable (WOL)
+ * - Reset (RST)
+ */
+struct MR {
+    /**
+     * \brief Register type.
+     */
+    using Type = std::uint8_t;
+
+    /**
+     * \brief Register offset.
+     */
+    static constexpr auto OFFSET = Memory_Offset{ 0x0000 };
```

Comment:
Change `OFFSET` to `MEMORY_OFFSET` and `Register offset` to `Register memory offset` (all register definitions).